pom.xml

```diff
@@ -19,94 +19,280 @@
-    </dependencies>
-  </dependencyManagement>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>addons-parent-pom</artifactId>
+        <groupId>org.exoplatform.addons</groupId>
+        <version>8-RC01</version>
+    </parent>
+
+    <groupId>org.exoplatform.addons.gamification</groupId>
+    <artifactId>gamification</artifactId>
+    <version>1.0.x-SNAPSHOT</version>
+    <packaging>pom</packaging>
+    <name>eXo Gamification</name>
+    <description>eXo Gamification extension</description>
+
+    <modules>
+        <module>services</module>
+        <module>portlets</module>
+        <module>extension</module>
+        <module>packaging</module>
+    </modules>
+
+    <scm>
+        <connection>scm:git:git://github.com/exo-addons/gamification.git</connection>
+        <developerConnection>scm:git:git@github.com:exo-addons/gamification.git</developerConnection>
+        <tag>HEAD</tag>
+        <url>https://github.com/exo-addons/gamification</url>
+    </scm>
+
+    <properties>
+        <!-- Defaut compiler settings -->
+        <maven.compiler.optimize>true</maven.compiler.optimize>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <!-- eXo Modules -->
+        <!-- **************************************** -->
+        <!-- Dependencies versions                    -->
+        <!-- **************************************** -->
+        <org.exoplatform.depmgt.version>13-RC01</org.exoplatform.depmgt.version>
+        <org.exoplatform.platform.version>5.0.0</org.exoplatform.platform.version>
+        <!-- for tests -->
+        <junit.version>4.12</junit.version>
+        <!-- Liquibase stack -->
+        <org.liquibase.version>3.5.5</org.liquibase.version>
+        <org.liquibase-hibernate4.version>3.6</org.liquibase-hibernate4.version>
+        <org.javassist.version>3.20.0-GA</org.javassist.version>
+        <validation-api.version>1.1.0.Final</validation-api.version>
+        <xml-apis.version>1.4.01</xml-apis.version>
+        <profile.liquibase/>
+        <!-- MySQL driver-->
+        <mysql.version>5.1.23</mysql.version>
+        <!-- Juzu Version -->
+        <juzu.version>1.1.0</juzu.version>
+        <!-- Lombok driver-->
+        <lombok.version>1.16.22</lombok.version>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <!-- Import versions of external dependencies to use -->
+            <dependency>
+                <groupId>org.exoplatform</groupId>
+                <artifactId>maven-depmgt-pom</artifactId>
+                <version>${org.exoplatform.depmgt.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
```

Comment:
You should not need to depend on depmgt directly. Depending on org.exoplatform.platform:platform should be sufficient. Did you try to remove it ?

Reply:
Sounds good without 
Done